htmd/trajectorystorage.py

```diff
@@ -0,0 +1,448 @@
+from natsort import natsorted
```

Comment:
Not related with this PR

Reply:
I have no clue what the hell happened in this PR. This is a totally different branch....